spec/models/links_udpate_spec.rb

```diff
@@ -69,12 +67,24 @@
     end
 
     it 'adds an error message' do
+      stub_request(:post, "https://publishing-api.test.gov.uk/lookup-by-base-path")
```

Comment:
There is a helper method to allow us to do this more succinctly:

```
publishing_api_has_lookups('/my-base-path' => 'my-content-id')
```

Without worrying with what it's actually doing behind the scenes.